src/vsr/replica_test.zig

```diff
@@ -1000,7 +1001,7 @@ const TestClients = struct {
             }
         }
 
-        const tick_max = 2_400;
+        const tick_max = 3_000;
```

Comment:
Whats the reason for the bump?

Reply:
The rng got permuted, and the new permutation takes a little longer to run one of the test cases.
Long-term rather than a `tick_max` counter, this should actually detect state changes across replicas and keep ticking until they stop. But that is not trivial so the counter will do for now.
We can set this counter as high as we like, but keeping it lower makes the tests faster.